src/ol_infrastructure/applications/open_metadata/__main__.py

```diff
@@ -0,0 +1,209 @@
+    instance_name=f"open-metadata-db-{stack_info.env_suffix}",
+    password=open_metadata_config.get("db_password"),
+    subnet_group_name=apps_vpc["rds_subnet"],
+    security_groups=[open_metadata_database_security_group],
+    storage=open_metadata_config.get("db_capacity")
+    or str(AWS_RDS_DEFAULT_DATABASE_CAPACITY),
+    engine_major_version="15",
+    tags=aws_config.tags,
+    db_name="open_metadata",
+    **defaults(stack_info)["rds"],
+)
+open_metadata_db = OLAmazonDB(open_metadata_db_config)
+
+open_metadata_db_vault_backend_config = OLVaultPostgresDatabaseConfig(
+    db_name=open_metadata_db_config.db_name,
+    mount_point=f"{open_metadata_db_config.engine}-open-metadata",
+    db_admin_username=open_metadata_db_config.username,
+    db_admin_password=open_metadata_config.get("db_password"),
+    db_host=open_metadata_db.db_instance.address,
+)
+open_metadata_db_vault_backend = OLVaultDatabaseBackend(
+    open_metadata_db_vault_backend_config
+)
+
+open_metadata_db_consul_node = Node(
+    "open-metadata-postgres-db",
+    name="open-metadata-postgres-db",
+    address=open_metadata_db.db_instance.address,
+    opts=consul_provider,
+)
+
+open_metadata_db_consul_service = Service(
+    "open-metadata-instance-db-service",
+    node=open_metadata_db_consul_node.name,
+    name="open-metadata-db",
+    port=open_metadata_db_config.port,
+    meta={
+        "external-node": True,
+        "external-probe": True,
+    },
+    checks=[
+        ServiceCheckArgs(
+            check_id="open-metadata-instance-db",
+            interval="10s",
+            name="open-metadata-instance-id",
+            timeout="60s",
+            status="passing",
+            tcp=open_metadata_db.db_instance.address.apply(
+                lambda address: f"{address}:{open_metadata_db_config.port}"
+            ),
+        )
+    ],
+    opts=consul_provider,
+)
+
+# Create a vault policy to allow dagster to get to the secrets it needs
+open_metadata_server_vault_policy = vault.Policy(
+    "open-metadata-server-vault-policy",
+    name="open-metadata-server",
+    policy=Path(__file__)
+    .parent.joinpath("open_metadata_server_policy.hcl")
+    .read_text(),
+)
+## Begin block for migrating to pyinfra images
+consul_datacenter = consul_stack.require_output("datacenter")
+
+# Pulumi k8s omd help stuff. Still struggling with syntax.
+
+kubeconfig = cluster_stack.require_output("kube_config")
+cluster_name = cluster_stack.require_output("cluster_name")
+VERSIONS = cluster_stack.require_output("versions")
+
+k8s_provider = kubernetes.Provider(
+    "k8s-provider",
+    kubeconfig=kubeconfig,
+)
+
+open_metadata_application = kubernetes.helm.v3.Release(
+    f"{cluster_name}-open-metadata-application-helm-release",
+    kubernetes.helm.v3.ReleaseArgs(
+        name="open-metadata-application",
+        chart="open-metadata-application",
+        version=VERSIONS["OPEN_METADATA"],
+        namespace="applications",
```

Comment:
Shouldn't this be `open-metadata`?

Reply:
Mike implemented a check_cluster_namespace method to help with this.